src/containers/Header/NotificationsTray/index.jsx

```diff
@@ -0,0 +1,74 @@
+import styles from './style.postcss';
+
+import React, { PureComponent } from 'react';
+import Cursor from 'immutable-cursor';
+import Config from 'domain/Config';
+import moment from 'domain/moment';
+import connect from 'domain/connect';
+import Icon from 'components/Icon';
+import Callout from 'components/Callout';
+
+class NotificationsTray extends PureComponent {
+  constructor() {
+    super();
+    this.state = { open: false };
+    this._getRef = this._getRef.bind(this);
+  }
+
+  _getRef(node) {
+    this._node = node;
+  }
+
+  render() {
+    const { notifications } = this.props;
+    return <Callout content={
```

Comment:
Passing HTML directly here to the prop makes the whole thing a little bit harder to read. It might be good to wrap this `content` thing in a function to simplify the HTML in the `render()` method.